aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/UnitTests/SessionControllerIndex.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Mvc;
+using Moq;
+using TestingControllersSample.Controllers;
+using TestingControllersSample.Core.Interfaces;
+using TestingControllersSample.Core.Model;
+using TestingControllersSample.ViewModels;
+using Xunit;
+namespace TestingControllerSample.Tests.UnitTests
+{
+    public class SessionControllerIndex
+    {
+        [Fact]
+        public void ReturnsARedirectToIndexHomeWhenIdIsNull()
+        {
+            var controller = new SessionController(null);
```

Comment:
nit: Use named parameters whenever you pass in `null` or `bool` values as its hard to figure out what the parameter type is otherwise